src/components/Chat/Chat.jsx

```diff
@@ -1,39 +1,95 @@
-import React from 'react';
-// import { useSelector } from 'react-redux';
+import React, { useState, useEffect, useRef } from 'react';
+import { useSelector } from 'react-redux';
 import {
   Wrapper,
   Header,
   HeaderContent,
   MessageBlock,
   MessageBarWrapper,
   MessageBar,
+  ButtonAddFile,
   Input,
-  Button,
+  ButtonSendMessage,
 } from './ChatStyled';
 import Icons from '../Icons/Icons';
-// import { getCountryName } from 'redux-store/AuthOperations/selectors';
+import {
+  getCountryName,
+  getNumberOfParticipants,
+} from 'redux-store/AuthOperations/selectors';
 
 export default function Chat() {
-  // const countryName = useSelector(getCountryName);
+  const countryName = useSelector(getCountryName);
+  const participants = useSelector(getNumberOfParticipants);
+
+  const [messageBlockHeight, setMessageBlockHeight] = useState(0);
+  const [value, setValue] = useState('');
+
+  // const fileInput = React.createRef();
+  const textAreaRef = useRef(null);
+
+  const handleChange = e => {
+    setValue(e.target.value);
+    console.log(value);
+  };
+
+  const isInputNotEmpty = value.trim() !== '';
+
+  const updateMessageBlockHeight = () => {
+    if (textAreaRef.current) {
+      setMessageBlockHeight(textAreaRef.current.scrollHeight);
+    }
+  };
+
+  useEffect(() => {
+    textAreaRef.current.style.height = 'auto';
+    textAreaRef.current.style.height = textAreaRef.current.scrollHeight + 'px';
+    updateMessageBlockHeight();
+  }, [value]);
+
+  useEffect(() => {
+    const handleResize = () => {
+      updateMessageBlockHeight();
+    };
+  
+    window.addEventListener('resize', handleResize);
+  
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
 
   return (
     <Wrapper>
       <Header>
         <HeaderContent>
-          <h5>Country Name</h5>
-          <p>members</p>
+          {!countryName ? <h5>Country Name</h5> : <h5>{countryName}</h5>}
+          {participants != null ? (
+            <p>{participants} members</p>
+          ) : (
+            <p>0 members</p>
+          )}
         </HeaderContent>
       </Header>
-      <MessageBlock></MessageBlock>
+      <MessageBlock style={{ height: messageBlockHeight + 'px' }}></MessageBlock>
       <MessageBarWrapper>
         <MessageBar>
-          <Button>
+          {/* <ButtonAddFile type="file" ref={fileInput}> */}
+          <ButtonAddFile>
             <Icons name="attach-file" fill="var(--color-grey-9)" size="24" />
-          </Button>
-          <Input type="text" placeholder="Type here" />
-          <Button>
-          <Icons name="send" fill="var(--color-grey-9)" size="24" />
-          </Button>
+          </ButtonAddFile>
+          <Input
```

Comment:
I've noticed you used MUI library across the project. Why didn't you use MUI input component here?